### ISSUE TITLE
getDate

### ISSUE THREAD

**Issue by MaximilianPichler** [sanjeev07/fullcalendar-reactWrapper#1]
Hey!
Your wrapper is working really great so far!
I've one question though: How do I get the current date?
(https://fullcalendar.io/docs/current_date/getDate/)

**Comment by sanjeev07** [comment]
Hi @MaximilianPichler thanks for asking this question . actually to access all the full calendar method like you said ".fullCalendar('getDate');  " or something else like ".fullCalendar( 'incrementDate', duration ) " etc . you need to do little hack here . 
1) use ref in your Fullcalender component . 
`<FullCalendar
     ref = 'abc'> 
`
or 
` <FullCalendar
          ref={(input) => { this.textInput = input; }} > 
`
2) now using ref you need to access the instance property of the wrapper component .
    so basically as documented in Fullcalendar getDate method Returns a Moment for the current 
    date of the calendar. and to do that with fullcalendar-reactwrapper . you just need to copy below 
   code segment . 
`let moment = this.refs.abc.instance.fullCalendar('getDate'); || this.textInput.instance.fullCalendar('getDate')` 
above code will give you the current date of the calendar . hope this will resolves your problem .

**Comment by MaximilianPichler** [comment]
Thank you very much! Worked perfectly

**Comment by Bakushin10** [comment]
@MaximilianPichler @sanjeev07 
Could you show me how you implement it please?

**Comment by MaximilianPichler** [comment]
In your Component you add a reference to your FullCalender like this:
```
<FullCalendar
  //your props
  ref={calendar => this.fullCalendar = calendar}
/>
```

Then you can access the calendar and its functions like this:
`let moment = this.fullCalendar.instance.fullCalendar('getDate');
`

**Comment by Bakushin10** [comment]
@MaximilianPichler Thanks!

I was actually hoping to get help on #6

**Comment by Bakushin10** [comment]
@MaximilianPichler 

```
                <FullCalendar {...options}
                    ref={calendar => {
                        this.fullCalendar = calendar
                        let moment = this.fullCalendar.instance.fullCalendar('getDate');
                        console.log(moment)
                    }}
                />
```

I get an error on instance
